modules/support-starting-an-alternative-image-with-toolbox.adoc

```diff
@@ -27,24 +33,19 @@ endif::openshift-origin[]
 # chroot /host
 ----
 
-. Create a `.toolboxrc` file in the home directory for the root user ID:
-+
-[source,terminal]
-----
-# vi ~/.toolboxrc
-----
+. Optional: If you need to use an alternative image instead of the default image, create a `.toolboxrc` file in the home directory for the root user ID, and specify the image metadata:
 +
 [source,text]
 ----
-REGISTRY=quay.io                <1>
-IMAGE=fedora/fedora:33-x86_64   <2>
-TOOLBOX_NAME=toolbox-fedora-33  <3>
+REGISTRY=quay.io             <1>
+IMAGE=fedora/fedora:latest   <2>
+TOOLBOX_NAME=toolbox-fedora-latest  <3>
 ----
-<1> Optional: Specify an alternative container registry.
+<1> This line is optional. Specify an alternative container registry.
 <2> Specify an alternative image to start.
-<3> Optional: Specify an alternative name for the toolbox container.
+<3> This line is optional. Specify an alternative name for the toolbox container.
```

Comment:
Should revert this change. `Optional:` is the standard we specify in the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks) and is used consistency in the repo (2000 hits vs under 100)
![image](https://github.com/user-attachments/assets/d34433fb-457b-42f3-8e4f-0775ad0e573a)  ![image](https://github.com/user-attachments/assets/2670468c-00dc-40fb-a5f0-5822eda2ce51)